cinema/views.py

```diff
@@ -2,0 +1,65 @@
+from rest_framework import viewsets
+
+from cinema.models import Actor, Genre, CinemaHall, Movie, MovieSession
+from cinema.serializers import (
+    GenreSerializer,
+    ActorSerializer,
+    CinemaHallSerializer,
+    MovieSerializer,
+    MovieSessionSerializer,
+    MovieListSerializer,
+    MovieRetrieveSerializer,
+    MovieSessionListSerializer,
+    MovieSessionRetrieveSerializer,
+)
+
+
+class GenreViewSet(viewsets.ModelViewSet):
+    queryset = Genre.objects.all()
+    serializer_class = GenreSerializer
+
+
+class ActorViewSet(viewsets.ModelViewSet):
+    queryset = Actor.objects.all()
+    serializer_class = ActorSerializer
+
+
+class CinemaHallViewSet(viewsets.ModelViewSet):
+    queryset = CinemaHall.objects.all()
+    serializer_class = CinemaHallSerializer
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.all()
+
+    def get_serializer_class(self):
```

Comment:
Consider adding type annotations to the methods in the viewsets to improve code readability and maintainability. For example, you can specify the return type of `get_serializer_class` as `def get_serializer_class(self) -> Type[serializers.Serializer]: ...`.